blockchain/genesis/genesis.go

```diff
@@ -169,6 +170,8 @@ type (
 		FairbankBlockHeight uint64 `yaml:"fairbankHeight"`
 		// GreenlandBlockHeight is the start height of storing latest 720 block meta and rewarding/staking bucket pool
 		GreenlandBlockHeight uint64 `yaml:"greenlandHeight"`
+		// HawaiiHeight is the start height to fix GetBlockHash in EVM
+		HawaiiHeight uint64 `yaml:"hawaiiHeight"`
```

Comment:
To be consistent with other height names, ```HawaiiBlockHeight```

Reply:
thx good catch